### ISSUE TITLE
Move the manifest.json from chrome/firefox branch to main branch

### ISSUE THREAD

**Issue by officialbishowb** [codecollabspace/chatgpt_citations#15]
Should move the manifest.json to the main branch and also update the Firefox version to manifest 3

**Comment by mcquenji** [comment]
what does firefox need for v3? you mentioned they need some additional info, right?

**Comment by officialbishowb** [comment]
Yes, but it seems that these are required when using specific permissions only.

**Comment by officialbishowb** [comment]
Nevermind they are required when trying to publish it

![image](https://github.com/codecollabspace/chatgpt_citations/assets/55999477/f1a0825d-ea74-4795-a972-29f92e514300)

**Comment by officialbishowb** [comment]
So fixed it for now. Lets see if it leads to any error in chrome with the next release (1.2)
